Makefile.core.mk

```diff
@@ -191,6 +191,7 @@ install: pre-install
 	cd helm/higress; helm dependency build
 	helm install higress helm/higress -n higress-system --create-namespace --set 'global.local=true'
 
+HIGRESS_LATEST_IMAGE_TAG ?= latest
```

Comment:
@johnlanni I don't know the latest Docker image sha tag for the Higress container.